banzai_nres/tests/test_extract.py

```diff
@@ -157,14 +157,13 @@ def five_hundred_square_image(maxflux, number_traces, trace_width, read_noise=10
     data += np.random.normal(0.0, read_noise, size=data.shape)
     uncertainty = np.sqrt(data + read_noise ** 2)
     wavelengths = np.ones_like(traces) * 5  # dummy wavelengths image that has values distinct from flux and traces.
-    image = NRESObservationFrame([EchelleSpectralCCDData(data=data, uncertainty=uncertainty, traces=traces,
-                                                         profile=profile, wavelengths=wavelengths,
-                                                         meta={'OBJECTS': 'tung&tung&none'},
-                                                         blaze={'id': np.arange(number_traces) + 1,
-                                                                'blaze': [np.ones(traces.shape[1]) for i in
-                                                                          range(number_traces)],
-                                                                'blaze_error': [np.ones(traces.shape[1]) for i in
-                                                                                range(number_traces)]},
-                                                         fibers={'fiber': np.arange(number_traces) + 1,
-                                                                 'order': np.arange(number_traces) + 1})], 'foo.fits')
+    image = NRESObservationFrame([CCDData(data=data, uncertainty=uncertainty, meta={'OBJECTS': 'tung&tung&none'})],
+                                 'foo.fits')
+    image.traces = traces,
```

Comment:
The comma at the end of this line seems out of place

Reply:
Though it's possible I just don't understand the new syntax

Reply:
I agree. Extra comma here seems bad

Reply:
Agreed. That was a typo that was causing weird issues that I have now fixed.